ldk

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+command=$1
+params=${@:2}
+cur_dir=${PWD##*/}
+project_dir="/var/www/$cur_dir"
+
+case $command in
+    install)
+        lnk="/usr/bin/ldk"
+        echo "Installing..."
+        if [ "$EUID" -ne 0 ]
+        then
+            echo "Failed! Be shure that you run install with 'sudo'"
+            exit 1
+        else
+            if [ -L $lnk ]
+            then
+            rm $lnk
+            fi
+            ln -s `realpath $0` $lnk
+            echo "Success! Symbolic link ${lnk} created."
+            exit 0
+        fi ;;
+
+    artisan)
+        tail="${project_dir}/artisan $params" ;;
+
+    composer)
+        tail="${*} --working-dir ${project_dir}" ;;
+
+    npm)
+        tail="npm --prefix=${project_dir} $params" ;;
+
+    gulp)
+        tail="gulp $params --cwd ${project_dir}" ;;
+
+    bower)
+        tail="bower $params --config.cwd=${project_dir}" ;;
+
+esac
+
+if [ "$tail" != "" ]
+then
+    docker-compose -f ~/Projects/laradock/docker-compose.yml exec --user=laradock workspace /bin/bash -c "$tail"
```

Comment:
Instead of `~/Projects/laradock/docker-compose.yml` this should be fetching the `docker-compose.yml` file based on the script's location. Maybe allow specifying the `docker-compose` config file but keep `docker-compose.yml` as default?